server/migrations/20220607213602-terra-token-threshold.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkUpdate('ChainNodes', {
```

Comment:
Need to declare the transaction explicitly with `queryInterface.sequelize.transaction`.